src/macaron/database/table_definitions.py

```diff
@@ -511,6 +511,12 @@ class Provenance(ORMBase):
     #: The provenance payload.
     provenance_payload: Mapped[InTotoPayload] = mapped_column(ProvenancePayload, nullable=False)
 
+    #: The name of the provenance asset.
+    provenance_asset_name: Mapped[str] = mapped_column(String, nullable=True)
+
+    #: The URL of the provenance asset.
+    provenance_asset_url: Mapped[str] = mapped_column(String, nullable=True)
+
```

Comment:
Since you are adding these two columns, I was thinking to revisit this ORM mapping. Do you think 
```python
    #: The release tag commit sha.
    release_commit_sha: Mapped[str] = mapped_column(String, nullable=True)
    #: The release tag.
    release_tag: Mapped[str] = mapped_column(String, nullable=True)
```
could be redundant since the `commit_sha: Mapped[str] = mapped_column(String, nullable=True)` should be the same as release tag?